src/App.jsx

```diff
@@ -1,8 +1,50 @@
 import './App.css'
+import Button from "./components/Button.jsx";
+import Product from "./components/Product.jsx";
+import Tile from "./components/Tile.jsx";
 
 function App() {
+   function logClickTextCollect() {
+    console.log('to the collection')
+   }
+  function logClickTextBags() {
+    console.log('shop all bags')
+  }
+  function logClickTextOrders() {
+    console.log('pre-orders')
+  }
+
   return (
-      <h1>Handbags & Purses</h1>
+      <>
+          <header>
+            <h1>Handbags & Purses</h1>
+            <nav>
+                <Button type={'button'} onClicked={logClickTextCollect} invisable={false}>to the collection</Button>
+                <Button type={'button'} onClicked={logClickTextBags} invisable={false}>shop all bags</Button>
+                <Button type={'button'} onClicked={logClickTextOrders} invisable={true}>pre-orders</Button>
+            </nav>
+          </header>
+          <main>
+              <Product textLabel={'Best seller'} imageSrc={'src/assets/bag_1.png'} imageAlt={'pink bag'} nameBag={'The handy bag'} price={'€400,-'}/>
```

Comment:
Hier heb je overal de src genoemd maar het is beter om de afbeeldingen te importeren aan het begin van je code wat ik in mijn eedere comment aangegeven heb. Dan hoef je hier alleen de naam de van de tassen tussen de acoolades te zetten, is een stuk overzichtelijker.

 <main>
              <Product textLabel={'Best seller'} imageSrc={bag1} imageAlt={'pink bag'} nameBag={'The handy bag'} price={'€400,-'}/>
              <Product textLabel={'Best seller'} imageSrc={bag2} imageAlt={'brown bag'} nameBag={'The stylish bag'} price={'€250,-'}/>
              <Product textLabel={'New collection'} imageSrc={bag3} imageAlt={'green bag'} nameBag={'The simple bag'} price={'€300,-'}/>
              <Product textLabel={'New collection'} imageSrc={bag4} imageAlt={'yellow bag'} nameBag={'The trendy bag'} price={'€150,-'}/>
          </main>